### PR TITLE
add arabic translation

### PR DESCRIPTION
**Pull request by Fcmam5**

### CODE REVIEW

**Comment by goanpeca** [comment]
@Fcmam5 thanks for the help, is this still incomplete? do you plan to add more translations to this PR ?

**Comment by Fcmam5** [comment]
@goanpeca you are welcome. No, not for the moment 😞 .

File: content/contents+ar.lr [hunk 1]
```diff
@@ -4,18 +4,18 @@ _template: home.html
 ---
 title: بيئات تطوير متكاملة لبايثون
 ---
-summary: 
+summary:
 ---
 description: BeeWare is a collection of projects that can be used to help develop, debug and launch Python software. Each tool follows the Unix philosophy of doing one thing well. Each tool can be used in isolation, or they can be chained together to provide a rich set of programming tools.
```
Comment (goanpeca) [inline content/contents+ar.lr]: What about this content :-p ?

File: content/contents+ar.lr [hunk 2]
```diff
@@ -110,9 +110,9 @@ title: Contribute!
 ----
 text: If all this sounds interesting, why not take a look at the `BeeWare projects </project/projects>`__ and see how you can use them when developing Python applications? Or, if you'd like to contribute, and you've never contributed to open source before, start with our `guide for first time contributors </contributing/first-time/>`__. If you feel ready to dive straight in, check out the `project pages </project/projects/>`__ or our guide on `what you can do to help </contributing/how/>`__.
```
Comment (goanpeca) [inline content/contents+ar.lr]: Or this :-p ?

File: content/contents+ar.lr [hunk 3]
```diff
@@ -152,9 +152,9 @@ title: ### Members
 ----
 text: BeeWare is supported by users like you! Every little bit helps: `click here to contribute </contributing/membership/>`__.
```
Comment (goanpeca) [inline content/contents+ar.lr]: Or this :-p ?

File: content/contents+ar.lr [hunk 4]
```diff
@@ -148,14 +147,14 @@ size: lg
...
-image: 
+image:
 ----
-image_alignment: 
+image_alignment:
 ---
-gutter_top: ### Many thanks to
+gutter_top: ### جزيل الشكر لـ
 ---
 gutter_bottom: ### [... and many more](/project/projects/)
```
Comment (goanpeca) [inline content/contents+ar.lr]: This is starting to look awesome :-)

How about `[... and many more]` text :-) ?

File: content/project/contents+ar.lr [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+---
+gutter:
+
+لنرحّب ببروتوس
+===========
+.. image:: /static/images/brutus-128.png
+
+بروتوس النحلة هو شعار مشروع BeeWare . هو نحلة عاملة مشغولة.
+
+Fun fact: a person who keeps bees is called an **apiarist** - a name that the BeeWare project uses to describe the core team. This also reflects the relationship that the core team wishes to have with the community - we're shepherds of the community, and we'll provide direction, but we know the real work is done by the community as a whole.
```
Comment (goanpeca) [inline content/project/contents+ar.lr]: Lets get this fun fact going and I think we should be good to go :-), thanks again for all the hard work!

Comment (Fcmam5) [reply content/project/contents+ar.lr]: Done, and I'm sorry I couldn't help more than that for the moment.

**Comment by Fcmam5** [comment]
@goanpeca I resolved #313 and translated some strings.

**Comment by goanpeca** [comment]
Looks good!, lets merge :-)

**Pull request merged**